apps/sms/views/conversation/style/compose_view.css

```diff
@@ -0,0 +1,111 @@
+.new .article-list {
```

Comment:
How do you feel if we create ```views/composer/style``` now and move this file to ```views/composer/style/composer.css```? And if in future patches someone thinks that some file is composer-only - we can move it there (kind of slowly split conversation-composer step by step :))

Just a bit of bikeshedding: I'm not sure what is the best name for composer related stuff, for view I think  ```ComposerView``` (```views/style/composer.css``` and ```views/js/composer.js```) is good enough, but maybe we can rename ```Composer``` component to distinguish it from view (not in this patch)?